### PR TITLE
HWI Examples: Updates, typos and simplification

### PR DESCRIPTION
**Pull request by jonathancross**
Nice project :-)

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -122,8 +121,8 @@ bitcoin-cli scantxoutset start '[{"desc":"sh(wpkh(xpub6DP8WTA5cy2qWzdtjMUpLJHkzo
 {"xpub": "xpub6DP9afdc7qsz7s7mwAvciAR2dV6vPC3gyiQbqKDzDcPAq3UQChKPimHc3uCYfTTkpoXdwRTFnVTBdFpM9ysbf6KV34uMqkD3zXr6FzkJtcB"}
 ```
 
-2. With this xpub it is possible  extract the relevant UTXOs using the 
-`scantxoutset` in Bitcoin Core (from 0.17).
+2. With this xpub it is possible  extract the relevant UTXOs using the
+`scantxoutset` RPC call in Bitcoin Core.
```
Comment (instagibbs) [inline docs/examples.md]: it's only in 0.17, still pertinent to tell users

File: docs/examples.md [hunk 2]
```diff
@@ -1,48 +1,47 @@
...
+The OS in this case is macOS v10.13.6  (Darwin Kernel Version 17.7.0). In Linux the
 "path" is shorter.
 
-## Useful xpubs to extract
+## Extracting xpubs
 
-Starting from version Bitcoin Core v.0.17, It is possible to to retrieve 
-the Unspent transaction outputs relevant for a set of [Output Descriptors][1] 
-From Bitcoin Core with the `scantxoutset` funcion.
+Bitcoin Core v0.17[¹](#bitcoin-core-version) allows you to retrieve the unspent transaction outputs (utxo)
```
Comment (achow101) [inline docs/examples.md]: Please don't use superscripts.

File: docs/examples.md [hunk 3]
```diff
@@ -138,8 +137,9 @@ bitcoin-cli scantxoutset start '[{"desc":"wpkh(xpub6DP9afdc7qsz7s7mwAvciAR2dV6vP
 ```
 
 
-[1]: https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md
-[2]: https://github.com/bitcoin/bips/blob/master/bip-0049.mediawiki
-[3]: https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki
+## Notes
+
+<a name="bitcoin-core-version" title="Note on Bitcoin Core version">1</a> "Bitcoin Core" refers to v0.17.x and later.
```
Comment (achow101) [inline docs/examples.md]: Please don't use html. Anchors like these should be doable with just markdown. Also, I'm not sure that we really need these anchors.

File: docs/examples.md (hunk 1 above)
Comment (jonathancross) [reply docs/examples.md]: Am I understanding this correctly that you want it repeated on each of the 3 lines that mention `scantxoutset`? (as it used to be)

The purpose of the footnote was to reduce redundancy, but I can revert if you like.

File: docs/examples.md (hunk 3 above)
Comment (jonathancross) [reply docs/examples.md]: > Please don't use html. Anchors like these should be doable with just markdown.

Unfortunatly GitHub Flavored Markdown [doesn't support footnotes](https://stackoverflow.com/a/25584585/1117929) (named anchors)

> Also, I'm not sure that we really need these anchors.

The idea was to reduce redundancy, but I can remove if preferred and just use inline links.

File: docs/examples.md (hunk 2 above)
Comment (jonathancross) [reply docs/examples.md]: I'll address this after we know if footnotes are being removed or not.

File: docs/examples.md (hunk 1 above)
Comment (instagibbs) [reply docs/examples.md]: maybe it's ok, fine with as-is I guess

File: docs/examples.md (hunk 2 above)
Comment (jonathancross) [reply docs/examples.md]: I went ahead and replaced the superscript numbers with normal numbers.

File: docs/examples.md (hunk 3 above)
Comment (jonathancross) [reply docs/examples.md]: @achow101 Do you want to leave this as-is or should I convert to inline links to the info?

Comment (achow101) [reply docs/examples.md]: Can you do inline links?

**Comment by jonathancross** [comment]
@instagibbs  @achow101 Anything else you'd like to see here?

**Comment by instagibbs** [comment]
LGTM?

**Comment by achow101** [comment]
ACK 621d2bfffdd3e2aba3a6b30f896316450e72ab3a

**Pull request merged**